tests/manager/models/BUILD

```
@@ -1 +1,5 @@
 python_tests(name="tests")
+
+python_test_utils(
+    name="test_utils",
+)
```

No new line at the end of the file!
Ref: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_195
```suggestion
)

```